app/controllers/users_controller.rb

```diff
@@ -6,7 +6,12 @@ class UsersController < ApplicationController
 
   def index
     @users = User.all
-    #@WTPusers = User.order('id ASC')
+    if params.has_key?(:committee)
+      #set WTPusers = to the users in that committee
+      @WTPusers = params[:committee].users
```

Comment:
I am a little confused on what is going on here. What functionality is this for?